packages/frontend/src/features/sqlRunner/components/ContentPanel.tsx

```diff
@@ -84,7 +84,17 @@ export const ContentPanel: FC<Props> = ({
             ref={wrapperRef}
         >
             <Tooltip.Group>
-                <Paper shadow="none" radius={0} px="md" py="sm" withBorder>
+                <Paper
+                    shadow="none"
+                    radius={0}
+                    px="md"
+                    py="sm"
+                    sx={(theme) => ({
+                        borderWidth: '0 0 1px 1px',
+                        borderStyle: 'solid',
+                        borderColor: theme.colors.gray[3],
+                    })}
```

Comment:
These are a little annoying but I didnt see another great way to get only the borders we want